docs/backups.rst

```diff
@@ -45,10 +45,14 @@ Drop your existing local database and restore from the backup::
 There will be a bunch of errors related to the fact that the production dump has roles which your
 local DB doesn't have. It's OK to ignore them.
 
-Migrate and update the CMS 'site' so that your local server uses production content::
+Migrate the database::
 
     $ workon serviceinfo
     $ ./manage.py migrate --noinput
+
+Next, we need to copy the production CMS content to your local site. The value 'localhost:8000' is
+special because the 'Domain Name' is hard-coded to that value in the Django admin ("Sites" app)::
```

Comment:
The backed up data appears to already have an entry in Sites for 'localhost:8000'.

Reply:
Yes, which is why the management command has to use that exact value, so as to match the value already in the database, IIUC.